### ISSUE TITLE
Extending a region with edges!

### ISSUE THREAD

**Issue by coevolutionlab** [pangenome/odgi#515]
Hi @AndreaGuarracino and @ekg ,
Many thanks for developing "odgi"!!

I needed suggestions on the following problem, and if you could provide some answers, that would be super helpful. 
1. I am interested in a **region X** of my pan-genome graph. 
2. I have identified the nodes in this region.
3. I want to extend the **region X** left and right until all the nodes in the original **region X** have a connecting node (in left and right direction)
 
Is it even feasible with the current Odgi set of tools ? Apologies if my issue sounds ridiculous. 
If this is doable, could you please suggest a way (combination of tools)? 

Thanks!! 
best,
Gautam

**Comment by ekg** [comment]
My experience has been that it's easy to collect the path ranges in the
subgraph and expand these using the range arithmetic that is implied by
what you've described. You could do this with awk and bedtools. Then you
use the new targets to extract a new subgraph.

**Comment by gaushi** [comment]
Yes, I used Erik's suggestions!! Please feel free to close this issue. Thank you!! 
Gautam
